src/plone/restapi/tests/test_blocks_deserializer.py

```diff
@@ -308,3 +308,82 @@ def test_deserialize_subblocks_transformers(self):
             "html"
         ]
         self.assertEqual(block, u"<div>This stays</div>")
+
+    def test_slate_internal_link_deserializer(self):
+        """test_internal_link_deserializer."""
```

Comment:
@tiberiuichim I'd remove the comment line here entirely.

Reply:
yeah... I copied from old code. I'll remove them